doc/SCNvim.txt

```diff
@@ -205,6 +205,10 @@ Browse SuperCollider documentation in nvim (see |scnvim-help-system| for more in
 >
   let g:scnvim_scdoc = 1
 
+Disable syntax colouring in the post window (on by default)
```

Comment:
The documentation can be moved to the section above under **Post window**, but other than that I think this is good to go!